### DISCUSSION TITLE
Daily Standup - Friday, April 14

### DISCUSSION THREAD

**Discussion by antonieto** [antonieto/ibm-dashboard#119] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - I managed to automate the task of posting our daily standup every week day
 - Almost done with the design of our DB with @Int-Angel 
 - Added linting rules to our project

**What are your priorities for the next 24 hours?**:
 - Add a linting github action to prevent un-styled code
 - Create tasks for DB Deployments

**Blockers**:
 - Not sure how we will automate db deployments lol

**Shoutouts**:
 - @Int-Angel for helping with DB design

**Comment by Int-Angel** [comment]
**What did you achieve in the last 24 hours?**:
 - I worked on the DB design with @antonieto 
 - I researched places to store the data sources for the project
 - worked on the quality plan

**What are your priorities for the next 24 hours?**:
 - Finished DB design

**Blockers**:
 - Not sure where to store the files yet

**Shoutouts**:
 - @antonieto for being sexy

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Added tasks for mockup and finished it
 - Researched Tremor's graphic components

**What are your priorities for the next 24 hours?**:
 - Create tasks for auth

**Blockers**:
 - Still have to learn about auth (want to check Passport)

**Shoutouts**:
 - @Int-Angel he is really handsome

**Reply by Int-Angel** [reply]
great job!
